ttm/releaser.py

```diff
@@ -81,15 +81,15 @@ def version_from_project(self):
         return self.iso_build_version(self.project.name, self.project.image_products[0].package,
                                       arch=self.project.image_products[0].archs[0])
 
-    def maxsize_for_package(self, package):
+    def maxsize_for_package(self, package, arch):
         if re.match(r'.*-mini-.*', package):
             return 737280000  # a CD needs to match
 
         if re.match(r'.*-dvd5-.*', package):
             return 4700372992  # a DVD needs to match
 
-        if re.match(r'livecd-x11', package):
-            return 681574400  # not a full CD
+        if re.match(r'livecd-x11', package) and not re.match(r'aarch64', arch):
```

Comment:
At this point let's make it more explicit:

```suggestion
        if re.match(r'livecd-x11', package) and arch == 'x86_64':
```

Reply:
OK